### ISSUE TITLE
Question Regarding ImageNet-R Accuracy in DiffTPT

### ISSUE THREAD

**Issue by gyuwonchoi** [chunmeifeng/DiffTPT#2]
Hi,
Thank you for sharing your code of your amazing work.

I implemented DiffTPT on the generated ImageNet-R dataset using your provided code. 
However, when I attempted to reproduce the results, I achieved a top-1 accuracy of 56.80, which is 2 lower than what is reported in your paper.

Could you please provide some insights or recipes for reproduction?

Additionally, I conducted these experiments using an NVIDIA RTX A6000 GPU.

Thank you.

**Comment by YTyangting** [comment]
> 您好，感谢您分享您出色工作的代码。
> 
> 我使用您提供的代码在生成的 ImageNet-R 数据集上实现了 DiffTPT。但是，当我尝试重现结果时，我获得了 56.80 的前 1 准确率，这比您论文中报告的准确性低 2。
> 
> 您能否提供一些关于复制的见解或食谱？
> 
> 此外，我使用 NVIDIA RTX A6000 GPU 和 Pytorch 1.13.0+cu117 进行了这些实验。
> 
> 谢谢。
Hi, have you solved this problem? I am also experiencing the same problem, and after adjusting the learning rate did not improve the experimental accuracy

**Comment by YTyangting** [comment]
> Hi, Thank you for sharing your code of your amazing work.
> 
> I implemented DiffTPT on the generated ImageNet-R dataset using your provided code. However, when I attempted to reproduce the results, I achieved a top-1 accuracy of 56.80, which is 2 lower than what is reported in your paper.
> 
> Could you please provide some insights or recipes for reproduction?
> 
> Additionally, I conducted these experiments using an NVIDIA RTX A6000 GPU and Pytorch 1.13.0+cu117.
> 
> Thank you.

Hi, have you solved this problem? I am also experiencing the same problem, and after adjusting the learning rate did not improve the experimental accuracy

**Comment by chunmeifeng** [comment]
Hi YT,

It might be caused by the diffusion model. I’ve also noticed there can be
slight variations in multiple generated images, but the differences
shouldn’t be that significant. Since generated images take up a lot of
memory, I’m no longer storing them.

Best

YTyangting ***@***.***> 于2024年11月4日周一 17:49写道：
